### PR TITLE
Added Polish translation

### PR DESCRIPTION
**Pull request by michalzkosmosu**

### CODE REVIEW

**Comment by michalzkosmosu** [comment]
What could be wrong?

**Comment by unbelievableflavour** [comment]
Ill check later. Thnx for the MR!

**Comment by michalzkosmosu** [comment]
Does anyone know what I can do to fix 'The Travis CI build failed'?

**Comment by unbelievableflavour** [comment]
You could copy paste the travis file from 1 one of my other projecten like snaptastic. Should be fine there.

File: po/extra/pl.po [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+msgid "Bart Zaalberg"
+msgstr "Bart Zaalberg"
+
+#: data/com.github.bartzaalberg.alias.desktop.in:8
+msgid "com.github.bartzaalberg.alias"
+msgstr "com.github.bartzaalberg.alia"
+
+#: data/com.github.bartzaalberg.alias.desktop.in:12
+msgid "shortcuts;alias;commands;aliases;bash;"
+msgstr "skróty klawiszowe;alias;komendy;aliasy;bash"
```
Comment (ryonakano) [inline po/extra/pl.po]:
```suggestion
msgstr "skróty klawiszowe;alias;komendy;aliasy;bash;"
```

Strings for `Keywords` entry in desktop files must end with a semicolon.
([The CI also complains about that](https://travis-ci.org/bartzaalberg/alias/builds/624589947#L387))

**Review by ryonakano — CHANGES_REQUESTED** [review]

File: po/extra/pl.po [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+msgid "Choose between light and dark with dark mode"
+msgstr "Wybieraj pomiędzy jasnym i ciemnym trybem"
+
+#: data/com.github.bartzaalberg.alias.appdata.xml.in:228
+msgid "Bart Zaalberg"
+msgstr "Bart Zaalberg"
+
+#: data/com.github.bartzaalberg.alias.desktop.in:8
+msgid "com.github.bartzaalberg.alias"
+msgstr "com.github.bartzaalberg.alia"
```
Comment (unbelievableflavour) [inline po/extra/pl.po:56]: Should be alias

**Pull request merged**